README.rst

```diff
@@ -7,8 +7,9 @@
 • `Chat (Community & Support) <https://dvc.org/chat>`_
 • `Tutorial <https://dvc.org/doc/get-started>`_
 • `Mailing List <https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&id=24c0ecc49a>`_
+• `VS Code Extension <https://marketplace.visualstudio.com/items?itemName=Iterative.dvc>`_
 
-|CI| |Maintainability| |Coverage| |DOI|
+|CI| |Maintainability| |Coverage| |VSCode| |DOI|
```

Comment:
VS Code

Reply:
is this text visible?

Reply:
okay, probably not :)

Reply:
@shcheklein It's not visible and won't work with whitespace. See https://github.com/iterative/dvc/pull/7859#discussion_r893955863

Reply:
Source from https://docutils.sourceforge.io/docs/user/rst/quickref.html#citations:

> Citation labels contain alphanumerics, underlines, hyphens and fullstop

Reply:
Never mind, it does work per https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions, so I must have made some mistake when trying it before. Updated now.